Lib/extractor/__init__.py

```diff
@@ -75,3 +75,65 @@ def extractUFO(
         raise ExtractorError(
             "There was an error reading the %s file." % format
         )
+
+
+def cmdline():
+    """
+    Extract one ore more fonts to UFO. Installed as command line script
+    `extractufo`.
+
+    Usage: extractufo font [font ...]
+    """
+    from sys import exit
+    from argparse import ArgumentParser
+
+    parser = ArgumentParser(
+        description="Extract data from font binaries and build UFO objects from them.",
+        epilog="Each resulting UFO will be saved as FONT_FILE.ufo(z) in the same directory as the original FONT_FILE.",
+    )
+    parser.add_argument('FONT_FILE', help='Input font path', nargs="+")
+    parser.add_argument('-m', '--ufo-module', choices=['ufoLib2', 'defcon'], help='Select the default library for writing UFOs (default: autodetect, prefer ufoLib2)')
+    parser.add_argument('-z', '--zip', action="store_true", help="Output UFO ZIP")
```

Comment:
`help="Output UFOZ"`

Reply:
Hm, "UFO ZIP" is actually how the specification calls it. :)

Reply:
Cool. I can't remember, and I am used to thinking UFOZ, but fine with UFO ZIP

Reply:
From https://unifiedfontobject.org/versions/ufo3/#storage-formats:
> UFO has two possible storage formats. The first, “UFO Package”, is a multi-file, file system directory structure. The second, “UFO ZIP”, is a [ZIP archive](https://en.wikipedia.org/wiki/Zip_(file_format)) containing the same directory structure as defined for UFO Package.

I'll leave this as-is then.